spec/system/task_spec.rb

```diff
@@ -1,24 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
+  let(:task_done) { create(:task, :done, project_id: project.id)}
```

Comment:
Factoryに`association`を設定すると`project_id`の指定は省略できます！
```suggestion
  let(:task_done) { create(:task, :done)}
```

Reply:
「 b78adfd で修正しました。」のように指摘事項には対応するコミットIDを半角スペース区切りで返答するとリンクになるので確認しやすいですね。

Reply:
コミット IDをつけるようにします。ありがとうございます!